data_api/snapshot_convertor/src/test/scala/uk/ac/wellcome/platform/snapshot_convertor/services/ConvertorServiceTest.scala

```diff
@@ -243,7 +282,8 @@ class ConvertorServiceTest
           privateBucketName = privateBucket.name,
           privateObjectKey = "doesnotexist.txt.gz",
           publicBucketName = publicBucket.name,
-          publicObjectKey = "target.txt.gz"
+          publicObjectKey = "target.txt.gz",
```

Comment:
Should v1 and v2 not have different output names?

Reply:
Yeah, will fix – although I think the fixture creates a different bucket for each test, so we _should_ be protected.

Reply:
Are you concerned that they will blat each other work in production? The snapshot converter reads the bucket and the key to write to from the SQS message sent by elasticdump, which in turn reads it from the SQS message generated by the snapshot_scheduler lambda. The terraform config for that lambda is [here](https://github.com/wellcometrust/platform/blob/master/data_api/terraform/service_snapshot_scheduler.tf) and is already ensuring that the snapshot converter ends up generating two different files.

Reply:
Yup. Thanks for clarifying.